src/util/signal-handler.js

```diff
@@ -3,7 +3,7 @@ import {forwardSignalToSpawnedProcesses} from './child.js';
 
 function forwardSignalAndExit(signal: string) {
   forwardSignalToSpawnedProcesses(signal);
-  process.exit(1);
+  throw new Error('Received SIGTERM, terminating.');
```

Comment:
Will that print a stacktrace on the console? That won't be helpful for the user.